apps/www/src/hooks/use-prefetch-pr-feedback.ts

```
@@ -0,0 +1,78 @@
+              feedbackData,
+            );
+            queryClient.setQueryData(
+              ["pr-feedback-kanban", threadId],
+              feedbackData,
+            );
+          }
+        }
+      } catch (error) {
+        console.error("Error prefetching PR feedback batch:", error);
+        // Remove failed thread IDs from prefetched set so they can be retried
+        threadIds.forEach((id) => prefetchedRef.current.delete(id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)